src/main/java/com/openshift/jenkins/plugins/ClusterConfig.java

```diff
@@ -62,13 +62,13 @@ public void setServerCertificateAuthority(String serverCertificateAuthority) {
                 .fixEmptyAndTrim(serverCertificateAuthority);
     }
 
-    public boolean isSkipTlsVerify() {
-        return skipTlsVerify;
+    public boolean isSkipTLSVerify() {
+        return skipTLSVerify;
     }
 
     @DataBoundSetter
-    public void setSkipTlsVerify(boolean skipTlsVerify) {
-        this.skipTlsVerify = skipTlsVerify;
+    public void setSkipTLSVerify(boolean skipTLSVerify) {
```

Comment:
The only other instance of Tls in the project is in main/resources/com/openshift/jenkins/plugins/ClusterConfig/config.jelly and it will be updated to TLS in the next push, good catch, thanks!

Reply:
I just want to confirm, you mean setting skipTlsVerify/skipTLSVerify via pipeline code in a job correct? Such as using the openshift.setServerUrl() ?

Reply:
well, it can be either freestyle or pipeline jobs wrt cluster config ... both type of jobs leverage them

Reply:
This has been updated in the current push, i had to revert some of the TLS to Tls for backward compatibility